en/Live preview update.md

```diff
@@ -1,61 +1,59 @@
-Live Preview is introduced to help you preview what you're writing on the go, as well as hide Markdown formatting.
+Obsidian 13 introduces a brand new editor engine that comes with several enhancements to your editing experience:
 
-See Live Preview in action:
+- Live Preview
+- Spellchecker
+- Text substitution
+- Vim mode
```

Comment:
Vim mode option already exists in the legacy editor so I wouldn't say it's a new enhancement